docs/content/en/docs/migrate/metrics-observe/_index.md

```diff
@@ -20,9 +20,7 @@ and Keptn evaluations.
 > **Note**
 The full SLO capabilities
 provided by Keptn v1 such as weighting and scoring
-are currently under development for Keptn.
-You can follow and participate in the design and implementation process at
-[GitHub Epic 1646](https://github.com/keptn/lifecycle-toolkit/issues/1646).
+have a first implementation in the [Analysis](https://lifecycle.keptn.sh/docs/implementing/slo/).
```

Comment:
html test will fail due to this but link will be valid after merge

Reply:
Not crazy about merging something that is failing htmltest.  In this case, why are you using the full path to the file rather than a relative path -- so something like (../../implementing/slo) -- I think that is the right number of ../ strings but would need to play with it.

Reply:
😨 true I will change it